smf/sled-agent/non-gimlet/config-rss.toml

```diff
@@ -101,7 +101,7 @@ bgp = []
 # You can configure multiple uplinks by repeating the following stanza
 [[rack_network_config.ports]]
 # Routes associated with this port.
-routes = [{nexthop = "192.168.1.199", destination = "0.0.0.0/0"}]
+routes = [{nexthop = "192.168.1.1", destination = "0.0.0.0/0"}]
```

Comment:
Is this change intentional?

Reply:
Nope, thank you. That's my local changes need to get things out the network. I keep meaning to make a precommit hook...

Reply:
Nope, thank you. That's my local changes need to get things out the network. I keep meaning to make a precommit hook...